lib/drivers/dexcomDriver.js

```diff
@@ -1003,6 +1008,8 @@ module.exports = function (config) {
                 progress(100);
                 cb(err, data);
               });
+            } else {
+              cb(err, data);
```

Comment:
Yes, so it wasn't really a regression - the error just wasn't being passed up the chain in this specific instance (no meter data).